configurations/manager/1TB_dataset.yaml

```diff
@@ -10,5 +10,5 @@ stress_read_cmd: ["cassandra-stress read cl=ONE n=268435456 -schema 'replication
                   "cassandra-stress read cl=ONE n=268435456 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=536870913..805306368",
                   "cassandra-stress read cl=ONE n=268435456 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=805306369..1073741824"]
 
-instance_type_db: 'i3en.2xlarge'
+instance_type_db: 'i3en.3xlarge'
```

Comment:
> Why did you choose this specific instance type?

It is the instance type used for 2TB run, so if you want to compare three configs; 0.5, 1 and 2TB you have to use the same instance type

> cost optimization

from 0.9 to 1.3 USD per instance, I doubt we run it so frequently we will notice the difference in our AWS bill, moreover, the beefier instance should acomplish the same task in shorter time negating the cost difference

> Also, you can consider an option to specify instance type on jenkins pipeline layer (not affecting existing configuration files)

This would be even more convenient change, do we have an example PR how to do that?

Reply:
Ah, I see that there is no separate argument for instance_type in Manager job.
But we can achieve the same result by setting required type `SCT_INSTANCE_TYPE_DB=<type>` in `extra_environment_variables` parameter

Reply:
ok, will take a look at it

Reply:
looks like `SCT_INSTANCE_TYPE_DB=<type>` works as expected
https://argus.scylladb.com/tests/scylla-cluster-tests/c4371f87-816c-4dce-9ca9-db289aeba30d
reverting changes on yaml files